roles/kubernetes/node/tasks/main.yml

```diff
@@ -152,7 +153,7 @@
     - kube-proxy
 
 - name: Cleanup kube-proxy leftovers from node
-  command: "{{ local_release_dir }}/hyperkube kube-proxy --cleanup"
+  command: "{{ bin_dir }}/hyperkube kube-proxy --cleanup"
```

Comment:
hyperkube binary being moved/renamed to `{{ bin_dir }}/kubelet` I guess we could use it instead

Reply:
see my comment [here](https://github.com/kubernetes-sigs/kubespray/pull/4098/files#r250643453)

Reply:
Another option would be to call this `kube-proxy --cleanup` command from the container launched by `kubeadm`, I'll need to dig a bit more to see if that's easy to implement

Reply:
added in 5b4a3518d3abdcfa4f8d41270eec7df3b81073e9